### PR TITLE
Restricted permissions for GitHub tokens.

### PR DESCRIPTION
**Pull request by felixxm**
See https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token.

### CODE REVIEW

File: .github/workflows/new_contributor_pr.yml [hunk 1]
```diff
@@ -4,6 +4,10 @@ on:
   pull_request_target:
     types: [opened]
 
+permissions:
+    issues: write
+    pull-requests: read
```
Comment (felixxm) [inline .github/workflows/new_contributor_pr.yml:9]: https://github.com/Deborah-Digges/new-pull-request-comment-action/blob/224c179a9e23f65ec50ff3240b8716369dc415d7/src/index.js#L9

File: .github/workflows/new_contributor_pr.yml [hunk 2]
```diff
@@ -4,6 +4,10 @@ on:
   pull_request_target:
     types: [opened]
 
+permissions:
+    issues: write
```
Comment (felixxm) [inline .github/workflows/new_contributor_pr.yml:8]: https://github.com/Deborah-Digges/new-pull-request-comment-action/blob/224c179a9e23f65ec50ff3240b8716369dc415d7/src/index.js#L69

**Comment by felixxm** [comment]
I wasn't sure how to restrict token permissions for `schedules.yml` :thinking:

**Comment by felixxm** [comment]
> contents: read

Added. We will see tomorrow if it works :smile:

**Review by apollo13 — APPROVED** [review]

**Comment by felixxm** [comment]
@apollo13 Thanks :+1:

**Pull request merged**